packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -487,6 +487,7 @@ export enum ConfigVariableType {
 }
 
 export enum ConfigVariablesGroup {
+  AWSSeSSettings = 'AWSSeSSettings',
```

Comment:
I have trouble Understanding the SeS casing, I would have understood AWSSESSetting even if ugly. Or AwsSesSettings (better). But harder to find logic in AWSSeSSettings